GETTING_STARTED.md

```diff
@@ -0,0 +1,416 @@
+# Getting started
+
+In this document you will find a step-by-step guide on how to get Eunomia running in a local Minikube cluster.
+
+The resources Eunomia, the GitOps controller, is watching and how it is done will be discussed in general.
+
+You will run two examples:
+- Eunomia acting upon local GitOpsConfig changes,
+- Eunomia watching for changes in a remote repository and acting upon them.
+
+## Prerequisites
+
+In order to build Eunomia, you'll need the following:
+
+- [Git](https://git-scm.com/downloads)
+- [Go 1.12+](https://golang.org/dl/)
+- [Docker](https://docs.docker.com/install/)
+- [Operator SDK v0.8.1](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md)
+- Access to a Kubernetes cluster
+  - [Minikube](https://kubernetes.io/docs/setup/minikube/) (for local development and testing)
+    - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  - [Minishift](https://www.okd.io/minishift/) (for local development and testing, optional)
```

Comment:
The shell script you're providing doesn't work with minishift. So this might get confusing.